ledger/participant-integration-api/src/main/scala/platform/apiserver/services/ApiSubmissionService.scala

```diff
@@ -252,6 +252,7 @@ private[apiserver] final class ApiSubmissionService private[services] (
       result: CommandExecutionResult
   )(implicit telemetryContext: TelemetryContext): Future[state.SubmissionResult] = {
     metrics.daml.commands.validSubmissions.mark()
+    logger.info("Submitting transaction to ledger")
```

Comment:
```suggestion
    logger.debug("Submitting transaction to ledger")
```
We used to have a message here, but it was removed because it is doubling the write pipeline noise without bringing anything to th table

Reply:
I disagree here. Time spent in interpretation is a pretty useful thing to see from logs even at INFO level.

Reply:
It is useful in development, sure. When you have a message like this in production though you are spamming logfile in a massive manner. We already have problem dealing with GCP dropping our logs and VMBC is also drowning in GB of data. We are being asked to lower the noise, not the opposite

Reply:
Fair enough, I can live with turning on DEBUG.

Reply:
done